nexfil.py

```diff
@@ -78,41 +78,29 @@
 gh_version = ''
 twitter_url = ''
 discord_url = ''
+author = ''
 found = []
 codes = [200, 301, 302, 403, 405, 410, 418, 500]
 home = getenv('HOME')
 loc_data = home + '/.local/share/nexfil/dumps/'
 
 def fetch_meta():
-    global gh_version, twitter_url, discord_url
+    global gh_version, twitter_url, discord_url, author
     try:
-        rqst = get('https://raw.githubusercontent.com/thewhiteh4t/nexfil/master/metadata.json', timeout=5)
```

Comment:
so this is how it works :
1. the metadata.json file is fetched from the repo
2. if it is able to fetch then its fine otherwise it uses the local metadata.json file
3. fetching the file from repo is required to `check the latest version` so if someone is using an older version of the tool, when the metadata is fetched the version checking function will notify the user that a newer version is available

Reply:
makes sense. i just don't want my osint tool calling out to a raw github url to load data if i can make it not do that :)

Reply:
I understand, if you have any alternate way of notifying the user about an update I am all ears